db_json_util.py

```diff
@@ -74,6 +76,63 @@ def main() -> None:
                     else:
                         print(response.text)
 
+# export data using the API
+        if args.export_data:
+            # check if path exists
+            if args.json_chains:
+                response = requests.get(args.url + '/all/chains')
+                if not os.path.exists(path_chains):
```

Comment:
`os` is not needed here, you can do `path_chains.exists()` directly since it's a `Path` object.